src/App.tsx

```diff
@@ -1,18 +1,111 @@
+import React, { useState, useEffect } from 'react';
+import { Routes, Route } from 'react-router-dom';
 import { PeoplePage } from './components/PeoplePage';
 import { Navbar } from './components/Navbar';
-
+import { HomePage } from './components/homePage';
+import { PageNotFound } from './components/PageNotFound';
```

Comment:
Use correct file names 
![image](https://github.com/mate-academy/react_people-table-advanced/assets/99403198/79c4abd1-bfdd-44a3-b42f-c4c675058c83)

Reply:
Not fixed